packages/storybook/stories/va-link.stories.jsx

```diff
@@ -153,14 +159,38 @@ Calendar.args = {
   text: `Add to calendar`,
 };
 
+export const Icon = VariantTemplate.bind(null);
+Icon.args = {
+  ...defaultArgs,
+  'href': '#',
+  'text': `National Cemetery Administration`,
+  'icon-name': 'mail',
+  'icon-size': 3,
+};
+
 const ReverseTemplate = ({ filename, filetype, href, reverse, text }) => {
```

Comment:
Does this new variation work with `reverse`? 

![Screenshot 2024-06-10 at 1 55 04 PM](https://github.com/department-of-veterans-affairs/component-library/assets/872479/28762331-b26b-4d2d-8828-1ef2f2c28431)

Reply:
Yes, it does.

Reply:
I've added an example to the reverse story as well.